packages/react/src/components/LightboxMediaViewer/README.md

```diff
@@ -64,28 +59,19 @@ DDS_LightboxMediaViewer=true
 
 ## Props
 
-| Name      | Required | Data Type | Default Value | Description                                                        |
-| --------- | -------- | --------- | ------------- | ------------------------------------------------------------------ |
-| `title`   | NO       | String    | null          | Title text for LightboxMediaViewer image                           |
-| `copy`    | NO       | String    | null          | Short description text for LightboxMediaViewer image               |
-| `image`   | YES      | Object    | null          | Image object for LightboxMediaViewer. see `Image` below for detail |
-| `open`    | YES      | Boolean   | null          | sets `true | false` whether the LightboxMediaViewer is open/close  |
-| `onClose` | NO       | Function  | null          | Run the function to do something on close                          |
+| Name    | Required | Data Type | Default Value | Description                                                        |
+| ------- | -------- | --------- | ------------- | ------------------------------------------------------------------ |
+| `title` | NO       | String    | null          | Title text for LightboxMediaViewer image                           |
+| `copy`  | NO       | String    | null          | Short description text for LightboxMediaViewer image               |
+| `image` | YES      | Object    | null          | Image object for LightboxMediaViewer. see `Image` below for detail |
+| `open`  | YES      | Boolean   | null          | sets `true | false` whether the LightboxMediaViewer is open/close  |
 
 ### Image
 
-| Name  | Data Type | Description                 |
-| ----- | --------- | --------------------------- |
-| `uri` | Object    | Urls for images             |
-| `alt` | String    | Alternative text for images |
-
-### uri
-
-| Name     | Required | Data Type | Description                                            |
-| -------- | -------- | --------- | ------------------------------------------------------ |
-| `uri.sm` | NO       | String    | Url for image that uses for 0 to 320px breakpoint.     |
-| `uri.md` | NO       | String    | Url for image that uses for 320px to 672px breakpoint. |
-| `uri.lg` | YES      | String    | Url for image that uses for 672px and up breakpoint.   |
+| Name           | Data Type | Description                 |
+| -------------- | --------- | --------------------------- |
+| `defaultImage` | String    | Source for images           |
+| `alt`          | String    | Alternative text for images |
```

Comment:
I would link to the Image component documentation, so we don't have to continually update this information if something changes over there.

Reply:
you mean remove this from README.md file? referring to Image documentation.

Reply:
Right, remove it, and link to the Image documentation in its place.

Reply:
Okay

Reply:
updated the PR.